pkg/storage/cacher.go

```diff
@@ -467,8 +467,8 @@ type cacheWatcher struct {
 
 func newCacheWatcher(resourceVersion uint64, initEvents []watchCacheEvent, filter FilterFunc, forget func(bool)) *cacheWatcher {
 	watcher := &cacheWatcher{
-		input:   make(chan watchCacheEvent, 10),
-		result:  make(chan watch.Event, 10),
+		input:   make(chan watchCacheEvent, 100),
```

Comment:
Then why not 50? I am not a fan of random value. So I asked since I am interested in how we choose the const in general. We probably should understand what kind of load we are trying to hold before giving up. I will try to figure it out. This is not a bkocker for this patch.